src/vector/defn.rs

```diff
@@ -162,11 +162,8 @@ impl<'a> Field<'a> {
 
     /// Get default field value.
     pub fn default_value(&'a self) -> Option<String> {
-        let rv = unsafe { gdal_sys::OGR_Fld_GetDefault(self.c_field_defn) };
-        if rv.is_null() {
-            return None;
-        }
-        Some(_string(rv))
+        let c_ptr = unsafe { gdal_sys::OGR_Fld_GetDefault(self.c_field_defn) };
```

Comment:
`c_ptr` is owned by `c_field_defn` so we don't need to free it here?

Reply:
Owned by `OGRFieldDefn`.